api/board/gpio.md

```diff
@@ -0,0 +1,115 @@
+GPIO API
+========
+
+The GPIO API supports digital I/O pins.
+
+The API object
+--------------
+GPIO functionality is exposed by the [`GPIO`](#gpio) object that can be obtained by using the [gpio() method of the `Board` API](./README.md/#gpio).
+
+Implementations MAY also support an explicit constructor that runs the [`GPIO initialization`](#init) algorithm.
+
+### Examples
+
+```javascript
+try {
+  var board = require("iot-board");
+
+  // Configure GPIO using the board
+  var gpio3 = board.gpio(3);  // GPIO input pin with default configuration.
+
+  // Configure GPIO using a constructor.
+  var gpio4 = new GPIO(4, board);
+  // If 'board' is the default board, it can be omitted.
+  // var gpio4 = new GPIO(4);
+
+  // Read pin values.
+  console.log(board.name + " GPIO pin 3 value: " + gpio3.value);
+  console.log(board.name + " GPIO pin 4 value: " + gpio4.value);
+
+  // Release the pins.
+  gpio3.close();
+  gpio4.close();
+
+  var gpio5 = board.gpio({ pin: 5, mode: "output", activeLow: true });
+  gpio5.write(0);  // activate pin 2
+  gpio5.close();
+
+  var gpio6 = board.gpio({pin: 6, edge: "any"});
+  // will notify on both 0->1 and 1->0 changes.
+  gpio4.onchange = function(value) {
+    console.log("GPIO pin 6 has changed; value: " + value);
+    gpio6.close();  // also removes listeners
```

Comment:
Be consistent. Ie. you could do "All comments start with capital and ends with punctuation mark or some kind"